add tutorial (#486)
radarlabs/documentation: docs/tutorials/building-better-location-permission-gathering.mdx

---
sidebar_position: 10
title: Building better location permission gathering 
---

import Tabs from "@theme/Tabs";
import TabItem from "@theme/TabItem";

This tutorial will guide you on using Radar's location permission manager to build a best-in-class user interface for collecting on-device location with the Radar [iOS SDK](/sdk/ios) and [Android SDK](/sdk/android). Achieving high opt-in rates for location permission helps your app deliver more value to users with location-based features.

> While iOS and Android expose their own location permission APIs, using them directly can be confusing and clunky. At Radar, we've developed a comprehensive location permission management system that not only wraps the OS features but also provides advanced state management, tracking, and information collection through various channels. This system offers developers a clean and easy interface to streamline their location permission gathering UI flow while leveraging powerful underlying capabilities.

Reviewer: ```suggestion
While iOS and Android expose their own location permission APIs, using them directly can be confusing and clunky. We provide a comprehensive location permission management system that not only wraps the OS features but also offers advanced state management, tracking, and information collection through various channels. This system offers developers a clean and easy interface to streamline their location permission gathering UI flow while leveraging powerful underlying capabilities.
```
